src/community/geogig/pom.xml

```diff
@@ -32,7 +32,7 @@
     <repository>
       <id>oracle</id>
       <name>Oracles Maven Repository</name>
-      <url>http://download.oracle.com/maven/</url>
+      <url>https://download.oracle.com/maven/</url>
```

Comment:
Dead repo

Reply:
'Allow edits from maintainers' is checked. Feel free to remove this.